### PR TITLE
connectToScramble

### PR DESCRIPTION
**Pull request by eprince1**
connect to scramble

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -22,35 +26,57 @@ function initialize(projectId, options) {
...
-    }
+      if (currVersion !== snapshotValue.version) {
+        (snapshotValue.actions || []).forEach(action => actionFunctions[action](snapshotValue));
+
+        localStorage.setItem('jellySyncVersion', snapshotValue.version);
+        currVersion = snapshotValue.version;
+      }
+    });
+  } catch (e) {
+    endpoint = await getEndpoint(pId);
```
Comment (avisprince) [inline index.js:75]: so does this just loop if it can't connect?

Comment (eprince1) [reply index.js:75]: actually technically no cause it doesn't call connect, its kinda doing nothing right now

Comment (eprince1) [reply index.js:75]: what should we do so it doesn't loop?

File: index.js [hunk 2]
```diff
@@ -22,35 +26,57 @@ function initialize(projectId, options) {
...
+    endpoint = await getEndpoint(projectId);
+  }
 
-  const ref = database.ref(`projects/${projectId}`);
-  connect(ref);
+  dbRef = database.ref(`project/${endpoint}`);
+  connect();
 
-  ref.onDisconnect(() => connect(ref));
+  ref.onDisconnect(() => connect());
```
Comment (avisprince) [inline index.js]: dbRef right?

**Review by avisprince — APPROVED** [review]

**Pull request merged**